Include missing core object in glossary (#48318)
kubernetes/website: content/en/docs/reference/glossary/pod-template.md

---

> title: Pod Template

Reviewer: ```suggestion
title: PodTemplate
```

id: pod-template
date: 2024-10-13
short_description: >

>   A Pod Template defines template for creating pods.

Reviewer: ```suggestion
  A template for creating Pods.
```

> aka: 

Reviewer: ```suggestion
aka:
  - pod template
```

tags:
- core-object

---

> An API object that defines a template for creating {{< glossary_tooltip text="pod" term_id="pod" >}}, typically used in higher-level controllers.

Reviewer: ```suggestion
An API object that defines a template for creating {{< glossary_tooltip text="pod" term_id="pod" >}}.
The PodTemplate API is also embedded in API definitions for workload management, such as 
{{< glossary_tooltip text="Deployment" term_id="deployment" >}} or
{{< glossary_tooltip text="StatefulSets" term_id="StatefulSet" >}}.
```

<!--more--> 

> Pod Templates provide a specification that includes metadata, labels, and a pod's desired state, which is utilized by controllers like {{< glossary_tooltip text="deployment" term_id="deployment" >}} and {{< glossary_tooltip text="statefulset" term_id="statefulset" >}} to create and manage multiple {{< glossary_tooltip text="replicas" term_id="" >}} of {{< glossary_tooltip text="pod" term_id="pod" >}}.

Reviewer: Try this:
```suggestion
Pod templates allow you to define common metadata (such as labels, or a template for the name of a
new Pod) as well as to specify a pod's desired state.
[Workload management](/docs/concepts/workloads/controllers/) controllers use Pod templates
to define and manage one or more {{< glossary_tooltip text="Pods" term_id="pod" >}}.
When there can be multiple Pods based on the same template, these are called
{{< glossary_tooltip term_id="replica" text="replicas" >}}.

Although you can create a PodTemplate object directly, you rarely need to do so.
```
